packages/vscode-extension/src/debug/constants.ts

```diff
@@ -35,4 +35,16 @@ export enum ProgrammingLanguage {
   typescript = "typescript"
 }
 
-export const skipNgrokConfigKey = "fx-resource-local-debug.skipNgrok";
\ No newline at end of file
+export const skipNgrokConfigKey = "fx-resource-local-debug.skipNgrok";
+
+const allAddressIPv4 = "0.0.0.0";
+const allAddressIPv6 = "::";
+const loopbackAddressIPv4 = "127.0.0.1";
+const loopbackAddressIPv6 = "::1";
```

Comment:
What if the machine does not have ipv6?

Reply:
Good question. Did not find any document on this. Tried disabling IPv6 on my Windows machine, the result is that `net.listen` can still listens on "::". I am not sure whether disabling IPv6 takes effect.
However, if `net.listen` throws an error, only `EADDRINUSE` will be considered as port in use. For any other errors, simply ignore.

Reply:
Added `resolve(false)` for any other errors:
```
      server.once("error", (err) => {
              if (err.message.includes("EADDRINUSE")) {
                resolve(true);
              } else {
                resolve(false);
              }
            })
```